docs/src/main/asciidoc/undertow-reference.adoc

```diff
@@ -29,3 +29,39 @@ jar files as defined by the Servlet spec.
 You can make use of the Undertow predicate language using an `undertow-handlers.conf` file. This file should be placed
 in the `META-INF` directory of your application jar. This file contains handlers defined using the
 link:http://undertow.io/undertow-docs/undertow-docs-2.0.0/index.html#predicates-attributes-and-handlers[Undertow predicate language].
+
+== CORS filter
+
+Quarkus comes with a CORS filter which can be enabled in the Quarkus configuration file, `application.properties`:
+
+[source, properties]
+----
+quarkus.http.cors=true
```

Comment:
I might miss something but I don't see this property in our config?

Reply:
It's defined in [HttpBuildConfig](https://github.com/quarkusio/quarkus/blob/master/extensions/undertow/runtime/src/main/java/io/quarkus/undertow/runtime/HttpBuildConfig.java).

Reply:
Ah OK, totally missed it, thanks!